### DISCUSSION TITLE
Connection problem

### DISCUSSION THREAD

**Discussion by telinha** [rustdesk/rustdesk#7763] [Q&A] [answered]
Hello, support.

Today I can't connect to the RustDesk network, I live in Brazil, would it be a regional problem?

![Captura de tela 2024-04-18 112456](https://github.com/rustdesk/rustdesk/assets/84255696/f4a8e197-00ae-457e-99eb-74c0b0222359)

I searched and didn't find any discussion about not being able to connect.

I appreciate the attention.

**Comment by rustdesk** [comment, marked as answer]
https://twitter.com/rustdesk/status/1780975322042867862
